File: src/Paiza/python/D101.py
num1,num2 = map(int,input().split())

even = num1 % 2
odd = num2 % 2

print(even)
print(odd)

if even == 0 and odd == 0:
    print("NO")
elif even != 0 and odd != 0:
    print("NO")
else:
    print("YES")

File: src/Paiza/D101.kt
package Paiza

fun main() {
    val num = readLine()!!.split(" ").map { it.toInt() }

    val even = num[0] % 2
    val odd = num[1] % 2

    if (even == 0 && odd == 0){
        println("NO")
    }else if (even != 0 && odd != 0){
        println("NO")
    }else{
        println("YES")
    }
}